aten/src/ATen/core/TensorBase.h

```diff
@@ -43,7 +43,6 @@ inline bool variable_excluded_from_dispatch() {
   // Please read the comment in `VariableFallbackKernel.cpp` about the background of this change.
   return true;
 #else
-  TORCH_INTERNAL_ASSERT_DEBUG_ONLY(!c10::impl::tls_local_dispatch_key_set().excluded_.has(DispatchKey::Autograd));
```

Comment:
Fun fact: the "has()" implementation actually creates a DispatchKeySet with the given key. In this case, the Autograd key which is NOT a runtime key. So it is only by change today that our assert was overly restrictive, and that the Autograd alias key is located just after the end of runtime keys marker that this has been running. With "Autograd" being effectively, the last indexable runtime key.

Since nothing creates such a runtime key today though, this assert can safely be removed as it cannot ever trigger.